SpatialGDK/Source/SpatialGDK/Public/Schema/ActorGroupMember.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/Component.h"
+#include "SpatialCommonTypes.h"
+#include "Utils/SchemaUtils.h"
+
+#include "WorkerSDK/improbable/c_worker.h"
+
+namespace SpatialGDK
+{
+using FActorLoadBalancingGroupId = uint32;
+
+// The ActorGroupMember component exists to hold information which needs to be displayed by the
+// SpatialDebugger on clients but which would not normally be available to clients.
+struct SPATIALGDK_API ActorGroupMember : AbstractMutableComponent
```

Comment:
Inheriting from AbstractMutableComponent was mostly to be used by the StaticComponentView. Since these data will have their own view, inheriting from it looks unecessary.